app/core/database_indexes.py

```
@@ -0,0 +1,294 @@
+        "game_sentiment",
+        "teams",
+        "players",
+        "games",
+    ]
+
+    for collection_name in collections:
+        try:
+            collection = db[collection_name]
+            await collection.drop_indexes()
+            results[collection_name] = True
+        except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)